CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ and this project adheres to
 - Improve Sentry integration for the LRS
 - Update handbook link to `https://handbook.openfun.fr`
 
+### Fixed
+
+- `make bootstrap` errors on M1 Macs
+
```

Comment:
We don't usually log changes related to the development environment or project that are not related to end-users concerns. So I would suggest to remove this. Note that the CHANGELOG check in the CI is allowed to fail :wink: